typings/index.d.ts

```diff
@@ -1217,14 +1217,14 @@ declare module 'discord.js' {
     public iconURL(options?: ImageURLOptions): string | null;
   }
 
-  export class PermissionOverwrites {
-    constructor(guildChannel: GuildChannel, data?: object);
+  export class PermissionOverwrites extends Base {
```

Comment:
Not related to this line, but to this file, update GuildChannel#permissionOverwrites types too

Reply:
Done.